### PR TITLE
add new guide section about organizing and managing open source events

### PR DESCRIPTION
**Pull request by anajsana**
This commit adds a new guide section about how to organize and managing open source events

### CODE REVIEW

**Comment by caniszczyk** [comment]
@anajsana can you add an Acknowledgement section at the end with all the folks that contributed to the original gdoc? It was a lot of folks! :)

File: content/en/guides/organizing-and-managing-open-source-events.md [hunk 1]
```diff
@@ -0,0 +1,773 @@
+---
+title: Organizing and Managing Open Source Events
+---
+
+Open source events are popular as much for the networking opportunities as for the skill building exercises and knowledge sharing. Events run the gamut from single focused internal talks and multitrack corporate external events to the international Linux Foundation conferences. While decades of experience shape these events, today a global pandemic is driving a hard shift from physical to virtual venues. The change is likely permanent as virtual venues are unlikely to disappear once the novel coronavirus is curtailed or conquered. There are already hints that the future will see a rise of hybrid versions.
```
[the remark this answers is not in the document]
Comment (anajsana) [reply content/en/guides/organizing-and-managing-open-source-events.md]: Thank you! :blush:

**Comment by anajsana** [comment]
hey, @caniszczyk do you have any idea of how can know the people who contributed to the original doc? I can't see anything when looking at the g.doc file :disappointed:

**Review by CsatariGergely — APPROVED** [review]

**Comment by caniszczyk** [comment]
@anajsana I added the rest of the folks who contributed, feel free to add their bio / link to twitter if you like

Thanks for getting this over the line, include in the next OSPOlogy newsletter :)?

**Pull request merged**